debugger/src/debug.c

```diff
@@ -320,7 +324,11 @@ static void on_watch_dragged_callback(GtkWidget *wgt, GdkDragContext *context, i
     gpointer userdata)
 {
 	/* string that is dragged */
+#if GTK_CHECK_VERSION(3, 0, 0)
+	gchar *expression = (gchar*)gtk_selection_data_get_data(seldata);
```

Comment:
The function ```gtk_selection_data_get_data()``` exists since gtk 2.14 and so could be used for both verions.

Reply:
I agree. Fixed.